src/client/components/LoginUser.jsx

```diff
@@ -0,0 +1,38 @@
+import axios from "axios"
+import { useState } from "react"
+import { useNavigate } from "react-router-dom"
+import UserForm from "./UserForm"
+
+function LoginUser({ apiUrl }) {
+    const navigate = useNavigate()
+    const [loginMessage, setLoginMessage] = useState('')
+    
+    const handleLogin = async ({ username, password }) => {
+      try {
+        const { data } = await axios.post(`${apiUrl}/user/login`, {
```

Comment:
it would be a good idea to create a helper function for your `post` `get` (and other request methods) so that you don't have to keep writing the options object :)

Reply:
Ah yes good point, thanks for the feedback! :)